client/app/caseDistribution/components/LeverHistory.jsx

```diff
@@ -186,10 +34,10 @@ const LeverHistory = (props) => {
             </th>
           </tr>
         </tbody>
-        <tbody>{history.map((entry, index) =>
+        <tbody>{leverHistoryTable.map((entry, index) =>
           <tr key={index}>
             <td className={styles.historyTableStyling}>{entry.created_at}</td>
-            <td className={styles.historyTableStyling}>{entry.user}</td>
+            <td className={styles.historyTableStyling}>{entry.user_id}</td>
```

Comment:
This is incorrect, use `user_css_id`
```suggestion
            <td className={styles.historyTableStyling}>{entry.user_css_id}</td>
```